tests/unit/records/test_schemas.py

```diff
@@ -38,31 +38,33 @@ def test_minimal_json(app, db, minimal_record):
     Record.create(minimal_record)
 
 
-def test_recid(app, minimal_record):
+def test_recid(app, db, minimal_record):
     """Test recid property."""
     # String instead of number
     minimal_record['recid'] = '123'
-    pytest.raises(ValidationError, Record.create, minimal_record)
+    exc = pytest.raises(ValidationError, Record.create, minimal_record)
+    assert (exc.value.message)
```

Comment:
This only checks if the exception message (python string) evaluates to True, which is always True for all non-empty strings ;)

The idea was to check if the content of message is accurate e.g. that 'recid' cannot be a string but a number etc.
Then in a test below 'resource type' has to be an object and not just a string. etc.

If you can't make it work due to Py2/Py3 dependencies, just remove those checks completely, i.e. just check if pytest.raises as before.